### PR TITLE
move some pipelines notebooks from 'unofficial' to 'official'

### PR DESCRIPTION
**Pull request by amygdala**

### CODE REVIEW

**Comment by amygdala** [comment]
All tests pass (a minor miracle) so I think this is now ready for review.  Moving a set of pipelines notebooks into 'official' from 'unofficial'.

File: ai-platform-unified/notebooks/official/pipelines/automl_tabular_classification_beans.ipynb [hunk 1]
```diff
@@ -0,0 +1,1040 @@
+{
```
Comment (ivanmkc) [inline ai-platform-unified/notebooks/official/pipelines/automl_tabular_classification_beans.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #6.    def classif_model_eval_metrics(
</pre><p>Not sure if this was intentional or not, but 'classif'?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

**Comment by ivanmkc** [comment]
If it passes tests and content review you should be good to go.

**Comment by ivanmkc** [comment]
I'd loop in @telpirion

**Review by ivanmkc — APPROVED** [review]
LGTM from testing standpoint

**Review by telpirion — APPROVED** [review]

File: ai-platform-unified/notebooks/official/pipelines/automl_tabular_classification_beans.ipynb (hunk 1 above)
Comment (amygdala) [reply ai-platform-unified/notebooks/official/pipelines/automl_tabular_classification_beans.ipynb:1]: It _was_ an intentional abbreviation of 'classification'...  😬

**Comment by amygdala** [comment]
Thanks!

**Pull request merged**